ADR/0039-spdx-support.md

```diff
@@ -0,0 +1,146 @@
+
+| SPDX Attribute                  | CycloneDX Attribute                                        |
+|---------------------------------|------------------------------------------------------------|
+| package.properties = [          | component.annotations = [                                  |
+|   {“name”: …, “value”: …}       |   {..., annotator: <tool>”:jsonencoded”                    |
+| ]                               | ]                                                          |
+
+
+#### Formulations
+CycloneDX formulations describe how the container was manufactured. In SPDX, Relationship elements can be used for the same purpose. All elements in SPDX have SPDXID attribute which is an element identifier unique in the whole SBOM document. Relationship element describes relation between two elements using their SPDXID and relationship type. Relationship type BUILD_TOOL_OF can be used to express the relationship of packages which were used to build the container.
+
+| SPDX Attribute                  | CycloneDX Attribute                                        |
+|---------------------------------|------------------------------------------------------------|
+| Formulations.components = [{}]  | Relationships = [{                                         |
+|                                 |  spdxElementId = <A-PACKAGE-ID>                            |
+|                                 |  relationshipType=BUILD_TOOL_OF                            |
+|                                 |  relatedSpdxElement=<ROOT-DOCUMENT-ID>                     |
+|                                 | }]                                                         |
+
+
+#### Metadata.tools
+The CycloneDX metadata.tools sub attributes that we are mostly interested in are the vendor and name elements. Information about the creation of the SPDX document can be stored into creationInfo. CreationInfo.creators element is basically a list of strings. There’s a vague specification about how it should be structured in the standard. Strings should be formatted in the following way: “<Attribute>: <Value>”. For example vendor should be stored as “Vendor: <vendor>”
+
+| SPDX Attribute                                 | CycloneDX Attribute                               |
+|------------------------------------------------|---------------------------------------------------|
+| Metadata.tools = [{“vendor”: “X”, “name”: “Y”] | CreationInfo.creators = [“Vendor: X”, “Tool: Y”]  |
```

Comment:
> There’s a vague specification about how it should be structured in the standard. Strings should be formatted in the following way: “<Attribute>: <Value>”

Do you have a reference for this?

> CreationInfo.creators = [“Vendor: X”, “Tool: Y”]

So the Vendor is not associated with the Tool? They're just two elements in the creators list?

Reply:
https://spdx.github.io/spdx-spec/v2.3/document-creation-information/#68-creator-field

Reply:
It has been added, just the snipped in this conversation is not updated